models/settings.go

```diff
@@ -41,6 +41,8 @@ type Settings struct {
 	MetricsResolutions MetricsResolutions `json:"metrics_resolutions"`
 
 	DataRetention time.Duration `json:"data_retention"`
+
+	SSHKey string `json:"ssh_key"`
```

Comment:
If we want to support users overwriting that file without API, then there is no point to storing it in the database.